vendor/k8s.io/kubernetes/staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -139,15 +139,19 @@ func (a *APIInstaller) getResourceKind(path string, storage rest.Storage) (schem
 	}
 
 	object := storage.New()
-	fqKinds, _, err := a.group.Typer.ObjectKinds(object)
+	fqKinds, unversioned, err := a.group.Typer.ObjectKinds(object)
 	if err != nil {
 		return schema.GroupVersionKind{}, err
 	}
 
 	// a given go type can have multiple potential fully qualified kinds.  Find the one that corresponds with the group
 	// we're trying to register here
 	fqKindToRegister := schema.GroupVersionKind{}
+	unversionedCandidate := schema.GroupVersionKind{}
 	for _, fqKind := range fqKinds {
+		if unversioned && fqKind.Group == "" {
```

Comment:
> Hence, we have two choice: 1) we can register Status under the build apigroup (sttts@adc5280) or 2) make the api installer happy with unversioned kinds as well.

Option 1 seems more correct.  I haven't fully considered the repercussions, but perhaps a followup issue is in order.

Does the serialized return include an `apiVersion`?  Do we know what it is?  `meta.k8s.io` or `build.openshift.io` seem good, but `""` seems less good.